src/pages/components/menu.mdx

```diff
@@ -44,15 +45,15 @@ components:
 
 ## How to use it
 
-### Select Menu
+### Arrow
```

Comment:
I know @austin94 feels pretty strongly that `Select` shouldn't even be on this page as it is represented as it's own component. I tend to agree with him. In this case, the "Button Menu" would just be renamed "Default" and raise to the top.

Reply:
that works for me.

Reply:
@jzempel Sounds good to me 👍

Reply:
I'm still down with the new direction but might be worth saying for posterity the rationale for why Henry and I had two examples on this page: Henry brought up concerns related to making a distinction between the "Menu" types on the Menu page since our Menus can serve different `roles`. When the trigger is a Button, it's a `menu` ([example](https://share.getcloudapp.com/xQuDxEw8)). When it's a Select, it's a `listbox`. ([example](https://share.getcloudapp.com/7KumkWzm)) We though it helpful to represent both.

Reply:
>  @austin94 feels pretty strongly that Select shouldn't even be on this page

+1 

And renaming to "Default" and raising to the top works 👍 .

Reply:
This is certainly a distinction that holds value. But unless we are intent on calling out the important subtlety, it has little to no meaning for our audience. At present, they are the beneficiaries of having Garden solve these intricate details for them.

Reply:
I think we should leave ourselves the space to pull back the lid and expose these kinds of details. But "showing two examples instead of one" doesn't necessarily shine light on the intension.

Reply:
All good, I just wanted to represent the journey in case we were curious.

Reply:
Yeah, helpful to know. 🙇